targets/stm32/mqtts/src/MQTTClientapp.c

```diff
@@ -0,0 +1,108 @@
+#include "main.h"
+#include "MQTTClientapp.h"
+
+#define MESSAGE_DELAY 1000
+#define OS_DELAY 100
+#define KEEP_ALIVE_INT 60
+
+void createMainfluxChannel(void)
+{
+    const char *_preId = "channels/";
+    const char *_postId = "/messages";
+    strcpy(mfTopic, _preId);
+    strcat(mfTopic, mfChannelId);
+    strcat(mfTopic, _postId);
+}
+
+void mqttClientSubTask(void const *argument)
+{
+    while (1)
+    {
+        if (!mqttClient.isconnected)
+        {
+            MQTTDisconnect(&mqttClient);
+            mqttConnectBroker();
+            osDelay(MESSAGE_DELAY);
+        }
+        else
+        {
+            MQTTYield(&mqttClient, MESSAGE_DELAY);
+            osDelay(OS_DELAY);
+        }
+    }
+}
+
+void mqttClientPubTask(void const *argument)
+{
+    const char *str = "{'message':'hello'}";
+    MQTTMessage message;
+
+    while (1)
+    {
+        if (mqttClient.isconnected)
+        {
+            message.payload = (void *)str;
+            message.payloadlen = strlen(str);
+
+            MQTTPublish(&mqttClient, mfTopic, &message);
+        }
+        osDelay(1000);
+    }
+}
+
+int mqttConnectBroker()
+{
+    int ret;
+
+    net_clear();
+    ret = net_init(&net);
+    if (ret != MQTT_SUCCESS)
+    {
+        printf("net_init failed.\n");
+        return -1;

```

Comment:
Please declare the error code with DEFINE somewhere, do not use magic numbers in the code.